complex/ft_main.c

```diff
@@ -561,5 +581,7 @@ int main(int argc, char **argv)
 out:
 	if (opts.dst_addr)
 		fts_close(series);
+	if(filename)
+		free(filename);
```

Comment:
need space after if.  filename may be set to optarg, so it cannot be unconditionally freed.